src/offices/_single.html

```diff
@@ -11,119 +11,117 @@
 
 {% block content_main %}
 
-    {% import "macros/activity-snippets.html" as activity_snippets with context %}
-    {% set activities_feed = activity_snippets.render(office.tags if office.tags, include_date_flag=true, number_columns=2) %}
-
     {% if office.title %}
         <h1>{{ office.title | safe }}</h1>
     {% endif %}
 
     {% if office.intro.text %}
-    <section class="block
-                    block__flush-top
-                    block__flush-bottom
-                    content-l
-                    content-l__main
-                    content-l__large-gutters">
-        {% set intro = office.intro %}
-        {% set show_subscription = intro.subscribe_form and intro.govdelivery_code %}
-        <section class="content-l_col
-                        {% if show_subscription -%}
-                            content-l_col-1-2
-                        {% else -%}
-                            content-l_col-1
-                        {%- endif %}
-                        ">
-            <p class="h3">{{ intro.text | safe if intro.text }}</p>
-        </section>
-        {% if show_subscription %}
+        <section class="block
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/9e8c5f4f6941141a3a2a468846d052f91a332aad